passport.md

```diff
@@ -144,35 +143,26 @@ PASSPORT_PUBLIC_KEY="-----BEGIN PUBLIC KEY-----
 <a name="upgrading-passport"></a>
 ### Upgrading Passport
 
-When upgrading to a new major version of Passport, it's important that you carefully review [the upgrade guide](https://github.com/laravel/passport/blob/master/UPGRADE.md).
+When upgrading to a new major version of Passport, it's important that you carefully review [the upgrade guide](https://github.com/laravel/passport/blob/{{version}}/UPGRADE.md).
```

Comment:
```suggestion
When upgrading to a new major version of Passport, it's important that you carefully review [the upgrade guide](https://github.com/laravel/passport/blob/master/UPGRADE.md).
```

version refers to the laravel framework version and not to the passport version.

Reply:
Thanks!